spritzle/alert.py

```diff
@@ -110,7 +110,7 @@ def register_handler(self, alert_type, handler):
                         if alert.category() & v:
                             handlers.update(self.handlers.get(k, []))
                     for handler in handlers:
-                        futures.append(asyncio.ensure_future(handler(alert)))
+                        futures.append(self.loop.create_task(handler(alert)))
```

Comment:
Could you change futures to tasks? This would be a better name now that we know it will create a task, not a future.

Reply:
Well, a task is a future. 😉 But point taken, tasks is a better name, will do.